From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Piotr Szczepanik <piter75@gmail.com>
Date: Mon, 23 Mar 2020 18:44:03 +0100
Subject: [ARCHEOLOGY] Fixed slow boot of rk3399 with mainline (#1852)

> X-Git-Archeology: - Revision aa1b639535858362b31f5a34780aa359407bd6e9: https://github.com/armbian/build/commit/aa1b639535858362b31f5a34780aa359407bd6e9
> X-Git-Archeology:   Date: Mon, 23 Mar 2020 18:44:03 +0100
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Fixed slow boot of rk3399 with mainline (#1852)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 75d0f64e3d75e7c34466871b9723ef1a238609d8: https://github.com/armbian/build/commit/75d0f64e3d75e7c34466871b9723ef1a238609d8
> X-Git-Archeology:   Date: Fri, 17 Apr 2020 21:30:37 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Switch rk3399 to u-boot v2020.04 (#1873)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 5a2b2c360b9c26ca4bd0d309af7cd3994fd08b7d: https://github.com/armbian/build/commit/5a2b2c360b9c26ca4bd0d309af7cd3994fd08b7d
> X-Git-Archeology:   Date: Sun, 03 May 2020 19:15:46 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Prepare rockchip64 for switch to mainline u-boot and switched ROCK Pi 4 (#1934)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 86abac1fd4e169712a44d245cf7adef4ee17c420: https://github.com/armbian/build/commit/86abac1fd4e169712a44d245cf7adef4ee17c420
> X-Git-Archeology:   Date: Tue, 26 Oct 2021 22:14:41 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Cleanup u-boot configurations for rockchip64 derivatives (#3150)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 71ff7b3b989dcb7e86b7e1c00a408a2f53744ea0: https://github.com/armbian/build/commit/71ff7b3b989dcb7e86b7e1c00a408a2f53744ea0
> X-Git-Archeology:   Date: Thu, 11 Nov 2021 23:49:52 +0100
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Switched rockchip64 u-boot to v2021.07 (#3233)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2813365dd25e3ad110936cbf014b95b38d7090ec: https://github.com/armbian/build/commit/2813365dd25e3ad110936cbf014b95b38d7090ec
> X-Git-Archeology:   Date: Mon, 07 Nov 2022 21:29:00 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Move known non working rockhip64 boards to previous boot loader (#4392)
> X-Git-Archeology:
---
 drivers/clk/rockchip/clk_rk3399.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/clk/rockchip/clk_rk3399.c b/drivers/clk/rockchip/clk_rk3399.c
index 7d31a9f22a..e82a545c96 100644
--- a/drivers/clk/rockchip/clk_rk3399.c
+++ b/drivers/clk/rockchip/clk_rk3399.c
@@ -1394,7 +1394,7 @@ static int rk3399_clk_probe(struct udevice *dev)
 	}
 #endif
 
-	if (init_clocks)
+	if (init_clocks || 1 == 1)
 		rkclk_init(priv->cru);
 
 	return 0;
-- 
Armbian

